Make it possible to specify a isSuggestable function in NameResolver

lukedirtwalker opened the pull request:
This makes it possible to find Variables as well in the
AstQuery::nameQuery function.
<a href='#crh-start'></a><a href='#crh-data-%7B%7D'></a>
<a href='https://www.codereviewhub.com/'><img src='http://www.codereviewhub.com/site/github-bar.png' height=40></a>
<a href='https://www.codereviewhub.com/dimitar-asenov/Envision/pull/176?approve=1'><img src='http://www.codereviewhub.com/site/github-approve.png' height=26></a>&nbsp;<a href='https://github.com/dimitar-asenov/Envision/pull/176'><img src='http://www.codereviewhub.com/site/github-refresh.png' height=26></a>
<a href='#crh-end'></a>

dimitar-asenov commented:
This pull request is now managed with CodeReviewHub. <a href='https://github.com/dimitar-asenov/Envision/pull/176'>Refresh</a>.<a href='#crh-update-none'></a>

lukedirtwalker committed:
Make it possible to specify a isSuggestable function in NameResolver

This makes it possible to find Variables as well in the
AstQuery::nameQuery function.
